src/reducers/petitions.js

```diff
@@ -15,6 +16,10 @@ export default function petitions (state = initialState, action) {
       return Object.assign({}, state,
         action.petitions, { isLoading: false }
       );
+    case UPDATE_CITY_FILTER_VALUE:
```

Comment:
For consistency; I think we should just use the `fetchPetitions action` and re-use `REQUEST_PETITIONS` and `RECEIVE_PETITIONS` actions. This is what will happen in the pagination